package.json

```diff
@@ -50,7 +50,7 @@
     "apis-helpers": "0.0.1"
   },
   "scripts": {
-    "test": "node_modules/mocha/bin/mocha test/integration",
+    "test": "mocha test/integration",
```

Comment:
You need to install globally. `npm` is smart enough to look through your `node_modules` for binaries used in the `scripts` section in `package.json`.